tests/docker/bionic/Dockerfile

```diff
@@ -60,6 +60,11 @@ RUN apt-get update \
     pkg-config \
     && rm -rf /var/lib/apt/lists/*
 
+# The following packages are required for validating Python files.
+# The version of Pylint is set to 2.4.4 to match CI.
+RUN python3 -m pip install \
+    packaging mypy pylint==2.4.4
+
```

Comment:
This fixes #5412 .

Reply:
I believe that this change is no longer required

[Relevant discussion](https://github.com/ARMmbed/mbedtls-test/pull/31#discussion_r799704216)

Reply:
I haven't removed these changes as yet. Do we want to ?. As I understand it from my discussion with @gilles-peskine-arm  this docker is not being used and will at some point be completely removed. I've been running my trial tests on this docker. So if it doesn't hurt, I would like to keep these changes. It's a stand alone commit, so can be knocked off.

Reply:
I'm ok with having these changes. But please be aware that Arm will not make any effort to maintain this Dockerfile, and we're likely to remove it soon. We maintain and use https://github.com/ARMmbed/mbedtls-test/tree/master/resources/docker_files .